dd-sdk-android/src/test/kotlin/com/datadog/android/error/internal/CrashReportsFeatureTest.kt

```diff
@@ -250,4 +249,64 @@ internal class CrashReportsFeatureTest {
         assertThat(endpointUrl).isSameAs(endpointUrl2)
         assertThat(serviceName).isSameAs(serviceName2)
     }
+
+    @Test
+    fun `will use the custom service name if provided`(forge: Forge) {
```

Comment:
If you follow my suggestion above, you only need to test the service name resolution in the CoreFeatureTest file

Reply:
But still I want to keep these tests as this will guarantee that the serviceName will be set in these features and their default value will be the one from the CoreFeature